### DISCUSSION TITLE
Correct behavior for `buildSrc` plugins configurations

### DISCUSSION THREAD

**Discussion by mochadwi** [Splitties/refreshVersions#533] [Q&A] [answered]
Does it correct behavior to do the following?

Here's my `plugins` config for `buildSrc` directory

<img width="634" alt="Screen Shot 2022-04-07 at 11 23 43 AM" src="https://user-images.githubusercontent.com/7057959/162119999-2a5df05a-8a58-4dcb-8e87-e9881b0617ad.png">

Somehow every `gradlew refreshVersions` the `versions.properties` adds the `kotlin-dsl-*` 

<img width="431" alt="Screen Shot 2022-04-07 at 11 23 27 AM" src="https://user-images.githubusercontent.com/7057959/162119992-391897f7-b021-40c9-888e-eee09aade415.png">

Due to the above behavior, warning shown in our builds:

```
WARNING: Unsupported Kotlin plugin version.
The `embedded-kotlin` and `kotlin-dsl` plugins rely on features of Kotlin `1.5.31` that 
might work differently than in the requested version `1.6.10`.
```

related: 

1. https://stackoverflow.com/questions/71446382/unsupported-kotlin-version
2. https://github.com/gradle/gradle/issues/13020

**Comment by jmfayard** [comment, marked as answer]
Gradle fixes itself the versions for those plugins, so I think we should filter them out, never add them to versions.properties
